imu_complementary_filter/CMakeLists.txt

```diff
@@ -56,6 +59,7 @@ install(DIRECTORY launch
   DESTINATION share/${PROJECT_NAME}
 )
 
+ament_export_libraries(complementary_filter)
```

Comment:
added to independently use `complementary_filter` in other projects

Reply:
Good catch, that was missing too.